.github/workflows/test.yml

```
@@ -36,3 +36,31 @@ jobs:
       run: |
         python -m coverage run --source packaging/ -m pytest --strict tests
         python -m coverage report -m --fail-under 100
+
+
+  test-pypy:
+    name: Test ${{ matrix.pypy_version }} on ${{ matrix.os }}
+    runs-on: $ {{matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        pypy_version: ['pypy2', 'pypy3']
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-python@v1
+      name: Install ${{ matrix.pypy_version }}
+      with:
+        pypy-version: ${{ matrix.pypy_version }}
+        architecture: x64
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -r dev-requirements.txt
+    - name: Run pytest
+      run: |
+        python -m pytest --capture=no --strict
```

We should probably stick with checking coverage here too:

```suggestion
        python -m coverage run --source packaging/ -m pytest --strict tests
        python -m coverage report -m --fail-under 100
```